### PR TITLE
Courses landing pages refresh, and more

### PR DESCRIPTION
**Pull request by morsssss**
Seeing a higher bounce rate for the main course landing page, and wanting to get feedback from teachers, I did a few things here:

* Changed language on main and teachers landing pages.
* Enhanced main landing page with bigger CTA buttons and the like.
* Added badge to final page.

I suspect that others may want to tweak my design, make sure my CSS isn't propagating to non-course pages, etc. I also added a new button feature to `card-grid.j2`, which you may or may not want! I thought I'd just make things look the way I wanted, and if I failed to follow an amp.dev best practice or two, you'd let me know what to change!

Two questions:
* In `advanced-course/conclusion-and-whats-next.md`, I tried to center the badge, but it didn't center. My impression is that the `center` class added by `align='center'` doesn't do anything. Am I doing this wrong?
* I was looking for a way to let people know about the badge on the main landing page. I didn't spot a great way to do that. Suggestions taken!

### CODE REVIEW

**Comment by morsssss** [comment]
Thanks, @matthiasrrohmer! @sebil, what do you think?

File: pages/content/amp-dev/documentation/courses/advanced-course/conclusion-and-whats-next.md [hunk 1]
```diff
@@ -13,6 +13,10 @@ Most importantly, stay curious. Think about how other popular websites work, and
 
 Of course, in these courses, we could only cover a portion of AMP’s powerful features. If you’re looking for more learning resources, there are more guides and tutorials available on the [official AMP website](../../../documentation/guides-and-tutorials/index.html). Additionally, the [AMP examples](../../../documentation/examples/index.html) section contains many examples of how to build modern websites and tackle common web development problems using AMP. Finally, the AMP library is constantly being developed and improved. Make sure to check back frequently for new features!
 
+Below is your shiny new AMP Developer badge. Feel free to copy it and save it. Then you can use it where you like to show that you've completed the AMP coursework! You can also download this badge in other formats, or in other snazzy designs, <a href="https://drive.google.com/drive/folders/1V0XSDCyRnJxxeX04FdoYW_pX4qP6xsWE" target="_blank">here</a>.
+
+{{ image('/static/img/courses/site/badge-shield-2x.png', 200, 200, layout='fixed', align='center', alt='Your AMP Developer badge') }}
+
```
[the remark this answers is not in the document]
Comment (morsssss) [reply pages/content/amp-dev/documentation/courses/advanced-course/conclusion-and-whats-next.md]: Thank you!

**Comment by morsssss** [comment]
Thanks! Yes, please make any changes that are necessary. I work with this site so little except to simply submit and modify markdown files, so I appreciate your offer to fix this up!

**Comment by sebil** [comment]
@googlebot I consent.

**Comment by sebil** [comment]
@morsssss You're very welcome!

I've done all the changes now and extended the image macro with a shadow option, to disable the shadow at eg. transparent images like the new badge.

Feel free to check everything again and if there is nothing else, I'll merge the PR.

**Comment by morsssss** [comment]
When I pull this code down, I see template directives output to the browser. I've seen this in the past, but I don't remember why that was happening or what I had to do to correct the issue :)

Screen Shot 2020-01-15 at 12 34 27 PM

Also, the `/documentation/courses/` route doesn't work for me. I have to use `/documentation/courses/index.html`.

So, if this looks good to you, I'm sure it will look good to me too. You don't have to wait for me to just set this live!

**Comment by sebil** [comment]
@googlebot I fixed it.

**Pull request merged**